### ISSUE TITLE
Incorrect bounding rects returned from pygame.draw.ellipse

### ISSUE THREAD

**Issue by charlesej** [pygame/pygame#1241]
The following code gives an example of an incorrect bounding rect returned from `pygame.draw.ellipse`.

```python
import pygame
import time

pygame.init()

rect = pygame.Rect((0, 0), (10, 10))
rect.center = (-3, -3)

screen = pygame.display.set_mode((200, 200))
b_rect = pygame.draw.ellipse(screen, pygame.Color('red'), rect)
pygame.display.flip()

print('bounding rect = {}'.format(b_rect))
for pos in ((x, y) for y in range(b_rect.h) for x in range(b_rect.w)):
    print('color {} at pos {}'.format(screen.get_at(pos), pos))

time.sleep(2)
pygame.quit()
```

---

The above code draws an ellipse completely off the screen, but the given output shows that the returned bounding rect incorrectly indicates that something was drawn on the screen. The color of each pixel in the bounding rect is printed to show that no part of the ellipse was drawn there.

```
pygame 2.0.0.dev3 (SDL 2.0.10, python 3.7.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
bounding rect = <rect(0, 0, 2, 2)>
color (0, 0, 0, 255) at pos (0, 0)
color (0, 0, 0, 255) at pos (1, 0)
color (0, 0, 0, 255) at pos (0, 1)
color (0, 0, 0, 255) at pos (1, 1)
```

---

System details:
- os: windows 10 (64bit)
- python: 3.7.2 (64bit) and 2.7.10 (64bit)
- pygame: 2.0.0.dev3 (SDL: 2.0.10) at 853804f70d3e82c72f623a3d3deb4b637200d3bb

**Comment by charlesej** [comment]
Test added with `@unittest.expectedFailure` decorator via #1242.

**Comment by charlesej** [comment]
Resolved by #1494.
